.evergreen.yml

```diff
@@ -230,63 +245,45 @@ functions:
       display_name: "source map"
 
   attach-storybook:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        - storybook-static/
-      remote_file: spruce/${task_id}/storybook/
-      bucket: mciuploads
-      content_type: text/html
-      permissions: public-read
-      display_name: storybook-
-  
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
-  attach-storybook-assets-css:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/storybook-static/assets/*.css"]
-      remote_file: spruce/${task_id}/storybook/assets/
-      bucket: mciuploads
-      content_type: text/css
-      permissions: public-read
-      display_name: storybook_assets_css_
-
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
-  attach-storybook-assets-js:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/storybook-static/assets/*.js"]
-      remote_file: spruce/${task_id}/storybook/assets/
-      bucket: mciuploads
-      content_type: application/javascript
-      permissions: public-read
-      display_name: storybook_assets_js_
-
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
-  attach-storybook-assets-map:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/storybook-static/assets/*.js.map"]
-      remote_file: spruce/${task_id}/storybook/assets/
-      bucket: mciuploads
-      content_type: application/json
-      permissions: public-read
-      display_name: storybook_assets_map_
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_files_include_filter:
+          ["storybook-static/*.html"]
+        remote_file: spruce/${task_id}/storybook/
+        bucket: mciuploads
+        content_type: text/html
+        permissions: public-read
+        display_name: storybook-
+        preserve_path: true
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_files_include_filter:
+          ["spruce/storybook-static/**/*.js", "spruce/storybook-static/**/*.mjs"]
+        remote_file: spruce/${task_id}/storybook/
+        bucket: mciuploads
+        content_type: application/javascript
+        permissions: public-read
+        display_name: storybook_assets_js_
```

Comment:
I have a huge pet peeve over the blank space inserted between the display name and the file name  🫠 I would not be opposed if you wanted to change or remove the display names (optional‼️)

<img width="289" alt="Screen Shot 2023-02-06 at 4 43 19 PM" src="https://user-images.githubusercontent.com/47064971/217094622-f59ae94c-dfb9-4f5d-b88e-b76353ce2be4.png">